diff --git a/src/app/question-show/question-show.component.html b/src/app/question-show/question-show.component.html
index a6420a7..b6bb708 100644
--- a/src/app/question-show/question-show.component.html
+++ b/src/app/question-show/question-show.component.html
@@ -2,7 +2,7 @@
   question-show works!
 </p>
 
-<form ng-submit="processForm()" class="solve-form">
+<form (ng-submit)="onSubmit()" #questionForm="ngForm">
   {{type}}
   <table>
     <tr>
@@ -22,6 +22,8 @@
   </table>
   <div class="text-center">
     <button type="submit">submit</button>
-    
+    <button type="button" class="btn btn-default" (click)="processForm(); questionFormForm.reset()">New Questions</button>
+
+
   </div>
 </form>
diff --git a/src/app/question-show/question-show.component.ts b/src/app/question-show/question-show.component.ts
index c658668..4fd2a46 100644
--- a/src/app/question-show/question-show.component.ts
+++ b/src/app/question-show/question-show.component.ts
@@ -29,14 +29,15 @@ export class QuestionShowComponent implements OnInit {
 
   private assignResults(results) {
     this.questions = results;
-    console.log(this.questions);
+    // console.log(this.questions);
   }
 
   protected changeRadioButton(value, questionId) {
     console.log(value);
-    console.log(questionId);
+    // console.log(questionId);
   }
   protected processForm() {
+    alert('got the form info');
     console.log('test');
     event.preventDefault();
   }
diff --git a/src/app/questions.service.ts b/src/app/questions.service.ts
index e48d5ea..cbf1993 100644
--- a/src/app/questions.service.ts
+++ b/src/app/questions.service.ts
@@ -71,11 +71,11 @@ export class QuestionsService {
     CategoryQuestions.questionText = results[1];*/
 
 
-    console.log(results);
-    console.log('===========');
-    console.log(results[0]);
-    console.log(results[1]);
-    console.log('===========');
+    // console.log(results);
+    // console.log('===========');
+    // console.log(results[0]);
+    // console.log(results[1]);
+    // console.log('===========');
 
     return categoryQuestions;
   }
